pkg/cloudprovider/aws/securitygroups.go

```diff
@@ -40,9 +40,9 @@ func NewSecurityGroupProvider(ec2api ec2iface.EC2API) *SecurityGroupProvider {
 	}
 }
 
-func (s *SecurityGroupProvider) Get(ctx context.Context, constraints *v1alpha1.Constraints) ([]string, error) {
+func (p *SecurityGroupProvider) Get(ctx context.Context, constraints *v1alpha1.Constraints) ([]string, error) {
```

Comment:
Why the s/s/p btw?

Reply:
The rest of the providers use `p` so just wanted to make them all uniform.